### DISCUSSION TITLE
Hide fields in list columns and filters

### DISCUSSION THREAD

**Discussion by ckruppe** [payloadcms/payload#10097] [Q&A]
Hi, maybe someone has some info how to get by this problem:

Right now i work on an Multi Tenant app. I have made some user roles: 'super-admin', 'admin, 'user.
My tenants should not know about the tenancy of my admin panel so i made the Tenant field on every collection only visible to the 'super-admin' and this role is also, thanks to an custom field, the only one that can create users with that role.

But now i am facing the problem that the tenant field is visible in the list filters and column config. I would like to have it there. But only for the 'super-admin'. The only thing i found was disabeling the field everywhere with these options:

```
admin: {
    disableListColumn: true,
    disableListFilter: true
},
```

Is there any possibillity to make it available only under an condition? As i did not found any possibillity for these options to set the boolean dynamic like its possible with hooks and conditions.

**Comment by NestorCanales** [comment]
Did you find or reach any solutions? I am basically building a similar site and had some partial success but still have some issues when setting up some restrictions.
I would love to see if we can connect and perhaps see if we can  help each other

**Reply by ckruppe** [reply]
Hey sorry for my late answer. I did not really come to an solution. I leaved it like that for now. So no one can see the tenant field in the list view. But if you need help i am happy to connect.
